management/user_test.go

```diff
@@ -80,11 +92,35 @@ func TestUser(t *testing.T) {
 		t.Logf("%v\n", uu)
 	})
 
-	t.Run("Delete", func(t *testing.T) {
-		err = m.User.Delete(auth0.StringValue(u.ID))
+	t.Run("GetRoles", func(t *testing.T) {
+		var roles []*Role
+		roles, err = m.User.GetRoles(auth0.StringValue(u.ID))
 		if err != nil {
 			t.Error(err)
 		}
+		t.Logf("%v\n", roles)
+	})
+
+	t.Run("AssignRoles", func(t *testing.T) {
+		roles := []*Role{r1, r2}
+		err = m.User.AssignRoles(auth0.StringValue(u.ID), roles...)
+		if err != nil {
+			t.Error(err)
+			t.Fatal(err)
```

Comment:
I think you need just one. Maybe stick to `t.Error`? Not sure why but in most tests I used `Error` unless without the test the following tests wouldn't be useful. Usually when followed by a `DeleteXXX` I avoid fatal errors as they wouldn't run and leave orphaned resources created in the auth0 account.